ipn/ipnlocal/network-lock.go

```diff
@@ -330,16 +332,17 @@ func (b *LocalBackend) NetworkLockStatus() *ipnstate.NetworkLockStatus {
 	defer b.mu.Unlock()
 
 	var nodeKey *key.NodePublic
+	var nlPriv key.NLPrivate
 	if p := b.pm.CurrentPrefs(); p.Valid() {
 		nkp := p.Persist().PublicNodeKey()
 		nodeKey = &nkp
+		nlPriv = p.Persist().NetworkLockKey
 	}
 
-	if b.tka == nil {
+	if b.tka == nil || nlPriv.IsZero() {
 		return &ipnstate.NetworkLockStatus{
```

Comment:
Its important that we display the profiles network-lock key when its not initialized, someone might need it when constructing a `tailscale lock init` invocation.

Can we update the logic to always return it in `NetworkLockStatus` if its set, rather than combining the check with `b.tka == nil` ?

Reply:
Oh yeah you're right. When I wrote this I didn't fully realize what b.tka == nil meant. Will fix
Good catch!